index.js

```diff
@@ -100,23 +104,30 @@ let rcl = {
 
   ActionLib: ActionLib,
 
+  NodeOptions: Node.NodeOptions,
+
   /**
    * Create a node.
    * @param {string} nodeName - The name used to register in ROS.
    * @param {string} namespace - The namespace used in ROS, default is an empty string.
    * @param {Context} context - The context, default is Context.defaultContext().
+   * @param {NodeOptions} options - The options to configure the new node behavior.
    * @return {Node} The instance of Node.
    */
-  createNode(nodeName, namespace = '', context = Context.defaultContext()) {
+  createNode(nodeName, namespace = '', context = Context.defaultContext(),
+    options = Node.NodeOptions.defaultOptions) {
+  
     if (typeof (nodeName) !== 'string' || typeof (namespace) !== 'string') {
       throw new TypeError('Invalid argument.');
     }
 
     let handle = rclnodejs.createNode(nodeName, namespace, context.handle());
     let node =  new rclnodejs.ShadowNode();
-
-    node.init(nodeName, namespace);
+	  node.handle = handle;
```

Comment:
`node.handle` is assigned in two places now: here and below the debug statement. Maybe just pass it into init instead and let the Node assign it internally?

Reply:
good catch. I'm not sure how I goofed that up. But clearly I did. I'll commit a fix. thx.

Reply:
fixed in 9e4b732

Reply:
fixed in 9e4b732